### PR TITLE
show staking amount warning for less than 15100 POKTs

### PR DESCRIPTION
**Pull request by immunity20**

### CODE REVIEW

**Review by iajrz — DISMISSED** [review]
Nice catch; I might've decided to change the method to `LT` instead of changing the amount, but it works either way. Thanks for the contribution!

**Comment by iajrz** [comment]
This will take a little bit of time to merge in due to work we're doing around the code check-in pipeline; thanks again for the contribution!

**Comment by immunity20** [comment]
I changed to LT function

File: app/cmd/cli/stake.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ The signer may be the operator or the output address.`,
 			return
 		}
 		am := types.NewInt(int64(amount))
-		if am.LTE(types.NewInt(15100000000)) {
+		if am.LTE(types.NewInt(15099999999)) {
```
Comment (andrewnguyen22) [inline app/cmd/cli/stake.go]: Why LTE here but LT above?

Comment (immunity20) [reply app/cmd/cli/stake.go]: Initially I had .LTE 15099999999 in both places but as per iajrz comment I update LTE to LT but forgot to do the same for the second place. I have updated

**Review by oten91 — APPROVED** [review]

**Pull request merged**